### ISSUE TITLE
Interactive element does not meet minimum size nor spacing - Leaflet map of states

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9679]
# Motivation

Siteimprove is reporting "Interactive element does not meet minimum size nor spacing". This is a WCAG 2.1 AA compliance issue related to the Leaflet map of states. Seems the Alaska and Hawaii links are the culprit.

# Siteimprove info

**Technical doc**
https://alfa.siteimprove.com/rules/sia-r113

Siteimprove link to existing issue
- https://my2.siteimprove.com/Inspector/998093/A11y/Page?pageId=24733563876&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r113/failed/cjFkZHdqZ3k=;NWVmMzE3YWQ=

Example
- https://admissions.uiowa.edu/counselors
- https://pharmacy.uiowa.edu/pharmd/dual_cert/palliative/certificate_graduates

**Interactive element does not meet minimum size nor spacing**
Issue
2.5.8: Target Size (Minimum)
Interactive elements, like buttons or links, should be large enough or be positioned, so they can easily be clicked or tapped by users.

Elements should be at least 24 by 24 pixels, or if smaller, be positioned with enough spacing to not intersect with other interactive elements.
